tests/sentry/tasks/integrations/github/test_language_parsers.py

```diff
@@ -578,6 +578,50 @@ def test_typescript_example(self):
             "SeeMoreCard",
         }
 
+    def test_javascript_functions_after_const(self):
+
+        patch = """
+          "@@ -305,9 +285,7 @@ export const Redacted Redacted\n   // Redacted.\n   // Redacted \n   const redacted = redacted;\n+  const redacted = redacted();\n+  // const redacted = true;\n+  const redacted = redacted()\n+    redacted; // Redacted\nconst \n@@ -165,24 +171,40 @@ export const RedactedRedactedRedactedRedactedRedacted
+
+          // Redacted
+@@44,38@@ var arrow2 = (a, ...b) => 0;
+    """
```

Comment:
if you combine all of these patches does it work too?

Reply:
i just combined the locally, and it does! should I combine them?

Reply:
why not

Reply:
done